SwiftAppTemplate/Dummy-Use&Delete/SettingsScreen.swift

```diff
@@ -20,8 +23,30 @@ struct SettingsScreen: View {
             VStack(spacing: 30){
                 
                 Header(text: "Settings", hasBackButton: true, onBackArrowClick: {dismiss()})
+         
```

Comment:
The below titleValueViews can be put in one struct for better clarity.

Reply:
fixed